Extensions/Content/Dnn.PersonaBar.Extensions/Components/Security/Utility.cs

```diff
@@ -185,23 +186,8 @@ public static string GetFileCheckSum(string fileName)
 
         public static string GetApplicationVersion()
         {
-            return DotNetNukeContext.Current.Application.Version.ToString(3);
```

Comment:
HI @bdukes , this because Evoq and Platform checked application version with different ways, and will get different versions in some releases, so here need to change to always get the version from platform assembly.